api/managementpb/dbaas/psmdb_cluster.proto

```diff
@@ -28,6 +28,8 @@ message PSMDBClusterParams {
   message ReplicaSet {
     // Requested compute resources.
     ComputeResources compute_resources = 1;
+    // Disk size in megabytes.
```

Comment:
Megabytes or mebibytes?

Using bytes (and SI units in general) would solve that problem :)

Reply:
Good point, I though that int64 won't be able to store that value, but checked again and looks like bytes is the best solution.